tests/appsec/appsec_utils.py

```diff
@@ -73,7 +73,7 @@ def appsec_application_server(
 
         try:
             print("Waiting for server to start")
-            client.wait(max_tries=100, delay=0.1)
+            client.wait(max_tries=120, delay=0.1, initial_wait=1.0)
```

Comment:
Increasing `max_tries` by `20` already increases the time the test will run by 2s, right? What's the difference between adding `initial_wait=1.0` and setting `max_tries` to `130`, then?

Reply:
The main difference is that with `initial_wait=1.0` we don't create any requests during the startup time, and given all the code (server and client) runs in the same machine, the load is decreased.